big_tests/tests/sm_SUITE.erl

```diff
@@ -1018,12 +998,16 @@ get_session_pid(UserSpec, Resource) ->
             {ok, C2SPid}
     end.
 
+get_user_alive_resources(UserSpec) ->
```

Comment:
If you introduce a new function like this, I'd rename the already existing one (`get_user_resources`) for clarity. Now we have:

- local `get_user_alive_resources` which calls remote `get_user_resources`
- local `get_user_resources` which calls remote `get_user_present_resources `

Is it really the best mix? ;)

Reply:
It is definitely not, but I didn't want to be the only one thinking about changing a ton of code lines (calls to `get_user_resources`), but now that you seem to agree with me, I'll do that :D